src/CommandFactory.ts

```diff
@@ -20,6 +21,17 @@ import {EndMotion} from './motion/EndMotion';
 import {ForwardCharMotion} from './motion/ForwardCharMotion';
 import {ForwardWordMotion} from './motion/ForwardWordMotion';
 
+var motions = {
```

Comment:
This is to allow me to call a motion based off the stored action from the keybinding. I could also just make a switch case in the method to handle this. I'm starting to lean a bit more to the switch case.